src/modules/account/components/AccountDetails/AccountOverview/AccountOverview.css

```diff
@@ -2,3 +2,13 @@
   padding-left: 0 !important;
   padding-right: 0 !important;
 }
+
+.bannerSwiper {
```

Comment:
Feels like this (.bannerSwiper) class should contain this 
```
    background-color: var(--color-ultramarine-blue);
    border-radius: var(--border-radius-standard);
    margin-top: 32px;
```
And not the .infoBanner classes.

To prevent the bullet navigation from going outside on 13inch displays 
<img width="1232" alt="image" src="https://github.com/LiskHQ/lisk-desktop/assets/8784876/3b3d50a1-4ae7-41f1-aef0-65010fd46770">